src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -132,6 +149,7 @@ ucs_status_t uct_ib_mlx5_get_cq(struct ibv_cq *cq, uct_ib_mlx5_cq_t *mlx5_cq)
     mlx5_cq->uar       = uct_dv_get_info_uar0(dcq.dv.uar);
 #endif
     mlx5_cq->dbrec     = dcq.dv.dbrec;
+    memset(&mlx5_cq->cq_decomp, 0, sizeof(uct_ib_mlx5_cq_decomp_t));
```

Comment:
why is it part of the cq structure?
seems it's a temp result of decompression

Reply:
Discussed verbally, pls reopen in case of questions.